.ci/docker-compose.override.yml

```diff
@@ -10,6 +10,7 @@ services:
     environment:
         - INTEGRATION=${INTEGRATION:-false}
         - SECURE_INTEGRATION=${SECURE_INTEGRATION:-false}
+        - ELASTIC_STACK_VERSION=$ELASTIC_STACK_VERSION
```

Comment:
Note for reviewer: without this the `ENV['ELASTIC_STACK_VERSION']` in Ruby code sees an empty String